src/meter/Meter.ts

```diff
@@ -24,8 +16,9 @@ const useMeter = createHook<MeterOptions, MeterHTMLProps>({
   compose: useBox,
   keys: METER_KEYS,
 
-  useProps(options, { "aria-valuetext": ariaValueText, ...htmlProps }) {
-    const { value, max, min, percent } = options;
+  useProps(options, htmlProps) {
+    const { value, max, min, ariaValueText } = options;
+    console.log("%c ariaValueText", "color: #99adcc", ariaValueText);
```

Comment:
remove these console logs please.

Reply:
I am about to push a change removing this & more